openff/toolkit/topology/_mm_molecule.py

```diff
@@ -98,10 +94,6 @@ def get_bond_between(self, atom1_index, atom2_index):
     def particle(self, index) -> int:
         return self.atom(index)
 
```

Comment:
We can delete all of these since `_mm_molecule` isn't in the public API.

Reply:
Nice catch, I totally forgot about them and it didn't occur to me that they're not in any API.